website/docs/gitops-templates/templates.mdx

```diff
@@ -240,10 +240,30 @@ spec:
         name: "{{ .params.CLUSTER_NAME }}"
 ```
 
-### Parameter metadata - `spec.params`
+
+## Parameters
 
 You can provide additional metadata about the parameters to the templates in the `spec.params` section.
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository. 
```

Comment:
- Would remove "Currently" - not required in technical docs
- Would be explicit - in the git repository, PR -> pull request
- Could optionally omit "and write to your git repository"

Reply:
Done. Thanks.